docs/sphinx/examples/python/quantinuum

```diff
@@ -0,0 +1,49 @@
+import cudaq
+
+# You only have to set the target once! No need to redefine it
+# for every execution call on your kernel.
+# To use different targets in the same file, you must update
+# it via another call to `cudaq.set_target()`
+cudaq.set_target("quantinuum")
```

Comment:
By default, this submits to the quantinuum syntax checker. This will be confusing for people since the result they get back is non-sense (the histogram is always 0, and we only care if it worked). Let's keep it for the synchronous execution but
1) add a comment that we are submitting to the syntax checker first
2) not print the histogram but instead print something comprehensive like "submission to syntax checker worked!" (if it actually did...)
3) set the machine to the emulator for the async execution  and add a comment that now we are submitting to the quantinuum emulator (cudaq.set_target('quantinuum', machine='H1-2E'))